momento-sdk/src/main/java/momento/sdk/ScsTopicClient.java

```diff
@@ -0,0 +1,132 @@
+package momento.sdk;
+
+import com.google.protobuf.ByteString;
+import grpc.cache_client.pubsub._PublishRequest;
+import grpc.cache_client.pubsub._TopicValue;
+import io.grpc.stub.StreamObserver;
+import java.util.concurrent.CompletableFuture;
+import javax.annotation.Nonnull;
+import momento.sdk.auth.CredentialProvider;
+import momento.sdk.config.Configuration;
+import momento.sdk.exceptions.CacheServiceExceptionMapper;
+import momento.sdk.responses.topic.SubscriptionState;
+import momento.sdk.responses.topic.TopicPublishResponse;
+import momento.sdk.responses.topic.TopicSubscribeResponse;
+
+public class ScsTopicClient extends ScsClient {
+
+  private final ScsTopicGrpcStubsManager topicGrpcStubsManager;
+  private final CredentialProvider credentialProvider;
+
+  public ScsTopicClient(
+      @Nonnull CredentialProvider credentialProvider, @Nonnull Configuration configuration) {
+    this.credentialProvider = credentialProvider;
+    this.topicGrpcStubsManager = new ScsTopicGrpcStubsManager(credentialProvider, configuration);
+  }
+
+  public CompletableFuture<TopicPublishResponse> publish(
```

Comment:
I think we should add javadocs to these methods as well since they will be the ones customers will be directly calling